postscripts/functions

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+SYSTEMD=no
+egrep -q "release [7-8]" /etc/redhat-release && SYSTEMD=yes
+[ -f /etc/lsb-release ] && grep -q Ubuntu /etc/lsb-release && SYSTEMD=yes
+export SYSTEMD
+
+# Preserve groups in case postscripts are run by hand later
+if echo ${GROUP:-UnDeFiNeD} | grep -q UnDeFiNeD
+then
+   if [ -f /opt/xcat/xcatgroups ]
+   then
+      GROUP=`cat /opt/xcat/xcatgroups`
+   fi
+else
+   if [ ! -f /opt/xcat/xcatgroups ]
+   then
+      echo $GROUP >/opt/xcat/xcatgroups
+   fi
+fi
```

Comment:
These aren't functions ... and they don't look like they are necessary for anything else below.
Suggest moving them to their own file or turn them into functions.